src/script_fmt/xm_sh.tera

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# {{ HEADER_AUTOGEN }}
+{% if abort_on_error %}# Set propagation of error to exit on first error
+set -e
+{% endif %}
+ROOT="{{ root }}"
+{% for group in srcs %}
+{% if group.file_type == 'verilog' %}xmvlog  -sv \
```

Comment:
I see you have support for verilog files, which is a great start. As bender is generally built to support vhdl files as well, and this simulator supports vhdl, it would be great to ensure compatibility here as well.

Reply:
We don't have any VHDL code internally to test with. If there is a public Bender repo with VHDL code and a simulation target, please refer me to it an I'll try to include the template.